consensus/istanbul/core/core.go

```diff
@@ -306,6 +313,28 @@ func (c *core) commit() error {
 	return nil
 }
 
+// GetAggregatedEpochValidatorSetSeal aggregates all the given seals for a the SNARK-friendly epoch encoding
+// to a bls aggregated signature and bitmap
```

Comment:
We don't actually use a bitmap here, do we?

Reply:
You're right!